superset/assets/src/components/StackTraceMessage.jsx

```diff
@@ -38,7 +38,13 @@ class StackTraceMessage extends React.PureComponent {
         >
           {this.props.message}
           {this.hasLink() &&
-          <a href={this.props.queryResponse.link}> (Request Access) </a>
+          <a
+            href={this.props.queryResponse.link}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            (Request Access)
```

Comment:
in the UI does this have correct spacing? i.e., is it

`my message(Request Access)` (bad, no space after message) or
`my message (Request Access)` (good)

Reply:
if not I'd suggest changing the line 

`(Request Access)` to
`{' (Request Access)'}`

Reply:
(I guess this should be done in apache, sorry I missed it there)

Reply:
Thanks Chris, I will send out a follow-up PR on open source.